examples/pytorch/tgn/train.py

```diff
@@ -0,0 +1,243 @@
+import dgl
+import torch
+import numpy as np
+from tgn import TGN
+from data_preprocess import TemporalWikipediaDataset,TemporalRedditDataset,TemporalDataset
+from dataloading import FastTemporalEdgeCollator,NegativeSampler,FastTemporalSampler,TemporalEdgeDataLoader,temporal_sort,TemporalSampler,TemporalEdgeCollator
```

Comment:
If you import a lot of things from a file, a suggested way is:
from utils import (AA, BB, CC,
                             DD, EE, FF)